charms/worker/k8s/charmcraft.yaml

```diff
@@ -161,6 +161,32 @@ config:
         CIDR to use for Kubernetes services. After deployment it is
         only possible to increase the size of the IP range. It is not possible to
         change or shrink the address range after deployment.
+    local_storage_enabled:
+      type: boolean
+      description: |
+        Enable local storage provisioning. This will create a storage class
+        named "local-storage" that uses the hostPath provisioner. This is
+        useful for development and testing purposes. It is not recommended for
+        production use.
+    local_storage_local_path:
+      type: string
+      default: ""
+      description: |
+        The path on the host where local storage will be provisioned. This
+        path must be writable by the kubelet. This is only used if
+        local-storage.enabled is set to true.
+    local_storage_reclaim_policy:
+      type: string
+      description: |
+        The reclaim policy for local storage. This can be either "Delete" or
+        "Retain". If set to "Delete", the storage will be deleted when the
+        PersistentVolumeClaim is deleted. If set to "Retain", the storage will
+        be retained when the PersistentVolumeClaim is deleted.
+    local_storage_set_default:
+      type: boolean
+      description: |
+        Set the local storage class as the default storage class. This is only
+        used if local-storage.enabled is set to true.
```

Comment:
Came to same something similar here:

1) as discussed lets stick to `snake_case` config names.  That's good
2) let's namespace this with the feature name:  `storage_local_*`

Reply:
We should be using dashes and not underscores, see https://juju.is/docs/sdk/styleguide#heading--naming

Reply:
naming is hard...

I now use the dashes and namespace it as `local_storage`
We still need to decide on a prefix as discussed in MM.

Reply:
no prefix, as discussed in the stand-up